robots/asr_twodof_description/urdf/TwoDofs.urdf

```diff
@@ -0,0 +1,142 @@
+            <geometry>
+				<mesh filename="package://example-robot-data/robots/simplecube_description/meshes/qb_meshes/dae/qbmove_frame_m.dae" />
+            </geometry>		
+        </visual>
+    </link>  
+	
+	<!-- # Second -->
+    <joint name="J2" type="revolute">
+        <origin xyz="0 0.089 0" rpy="0 0 0"/>
+        <parent link="Link1"/>
+        <child link="Link2"/>
+		<limit effort="30" velocity="1.0" lower="-3.14" upper="3.14" />
+        <axis xyz="0 0 1"/>
+    </joint>
+    
+    <link name="Link2"> 
+        <inertial>
+            <origin xyz="0 0.085 0" rpy="0 0 0"/>
+            <mass value="0.55"/>
+            <inertia ixx="0.001" iyy="0.001" izz="0.001" ixy="0" ixz="0" iyz="0"/>
+        </inertial>	
+        <visual>
+            <origin xyz="0 0.056 0" rpy="0 -1.57 -1.57"/>
+            <geometry>
+                <mesh filename="package://example-robot-data/robots/simplecube_description/meshes/qb_meshes/dae/qb_c_flange_m.dae" />
+            </geometry>		
+        </visual>
+		<visual>
+            <origin xyz="0 0.089 -0.033" rpy="0 0 0"/>
+            <geometry>
+				<mesh filename="package://example-robot-data/robots/simplecube_description/meshes/qb_meshes/dae/qbmove_frame_m.dae" />
+            </geometry>	
+        </visual>
+    </link>  
+
+	<!-- # End effector -->
+    <joint name="EE" type="fixed">
+        <origin xyz="0 0.1220 0" rpy="0 0 0"/>
+        <parent link="Link2"/>
+        <child link="Tip"/>
+        <axis xyz="0 0 1"/>
+    </joint>
+	<link name="Tip"> 
+	</link> 
+	
+    <gazebo>
+        <self_collide>1</self_collide>
+    </gazebo>
+
+    <!-- Legend for the libadvanced_plugin.so custom plugin 
+    
+        # The type of actuator is fixed i.e., qbMove
+        # The control_type can be chosen as: [0,1,2]
+        #      - [0] -> references are directly the motor positions
+        #      - [1] -> references are used as equilibirium position and preset
+        #      - [2] -> first reference is used as direct joint torque, while the second is not used (marked as disabled)
+        #
+        # The three other tags enable the generation and publication of additional topics: the elastic topic publisher, the state publisher and 
+        #      enable the subscription to external torque topic
+
+    -->
+    <gazebo>
+        <plugin name="advanced_vsa_j1" filename="libadvanced_plugin.so">
+            <joint>J1</joint>
+            <control_type>1</control_type>
+            <pub_eltau>false</pub_eltau>
+            <pub_state>false</pub_state>
+            <sub_ext_tau>false</sub_ext_tau>
+        </plugin>
+    </gazebo>
+    <gazebo>
+        <plugin name="advanced_vsa_j2" filename="libadvanced_plugin.so">
+            <joint>J2</joint>
+            <control_type>2</control_type>
+            <pub_eltau>false</pub_eltau>
+            <pub_state>false</pub_state>
+            <sub_ext_tau>false</sub_ext_tau>
+        </plugin>
+    </gazebo>

```

Comment:
I think we should remove these plugins as they are not standard ones.
@nim65s  and @jcarpent - do you agree or not on this topic?